modules/azure-firewall/firewall.tf

```diff
@@ -0,0 +1,14 @@
+resource "azurerm_firewall" "main" {
+  count               = length(var.firewall)
+  name                = "${var.project}-${var.env}-${lookup(element(var.firewall, count.index), "location")}"
+  location            = azurerm_resource_group.main[count.index].location
+  resource_group_name = azurerm_resource_group.main[count.index].name
+
+  ip_configuration {
+    name                 = "configuration"
```

Comment:
shouldn't this come from a configuration value?
i.e. the traffic going to this IP is what is forwarded on, 
in this case it's AKS but their could be IP configurations?

Reply:
yeah, you are right. I intend to change it but somehow forgot about it.